src/data/screenshots_de.json

```
@@ -142,116 +156,139 @@
                 ]
             },
             {
                 "title": "Kontakt-Tagebuch",
                 "anchor": "ios_journal",
                 "images": [
-                    { "filename": "/assets/screenshots/2-3/de/ios/iPhone 11 Pro-contact_journal_information_screen_0001", "alt": "Kontakt-Tagebuch Bild 1 von 8" },
-                    { "filename": "/assets/screenshots/2-3/de/ios/iPhone 11 Pro-contact_journal_information_screen_0002", "alt": "Kontakt-Tagebuch Bild 2 von 8" },
-                    { "filename": "/assets/screenshots/2-3/de/ios/iPhone 11 Pro-contact_journal_information_screen_0003", "alt": "Kontakt-Tagebuch Bild 3 von 8" },
-                    { "filename": "/assets/screenshots/2-3/de/ios/iPhone 11 Pro-contact_journal_listing1_0001", "alt": "Kontakt-Tagebuch Bild 4 von 8" },
-                    { "filename": "/assets/screenshots/2-3/de/ios/iPhone 11 Pro-contact_journal_listing_add_persons", "alt": "Kontakt-Tagebuch Bild 5 von 8" },
-                    { "filename": "/assets/screenshots/2-3/de/ios/iPhone 11 Pro-contact_journal_listing_add_locations", "alt": "Kontakt-Tagebuch Bild 6 von 8" },
-                    { "filename": "/assets/screenshots/2-3/de/ios/iPhone 11 Pro-contact_journal_listing_edit_persons", "alt": "Kontakt-Tagebuch Bild 7 von 8" },
-                    { "filename": "/assets/screenshots/2-3/de/ios/iPhone 11 Pro-contact_journal_listing_edit_locations", "alt": "Kontakt-Tagebuch Bild 8 von 8" }
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-contact_journal_information_screen_0001", "alt": "Kontakt-Tagebuch Bild 1 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-contact_journal_information_screen_0002", "alt": "Kontakt-Tagebuch Bild 2 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-contact_journal_information_screen_0003", "alt": "Kontakt-Tagebuch Bild 3 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-contact_journal_listing1_0001", "alt": "Kontakt-Tagebuch Bild 4 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-contact_journal_listing_add_persons", "alt": "Kontakt-Tagebuch Bild 5 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-contact_journal_listing_add_locations", "alt": "Kontakt-Tagebuch Bild 6 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-contact_journal_listing_edit_persons", "alt": "Kontakt-Tagebuch Bild 7 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-contact_journal_listing_edit_locations", "alt": "Kontakt-Tagebuch Bild 8 von 8" }
                 ]
             },
             {
-                "title": "Check-in",
+                "title": "Check-In - Scan QR-Code",
```

```suggestion
                "title": "Check-in - QR-Code scannen",
```

Why is this section only there for iOS?

---

Hi, thanks. We don't have the corresponding screenshots for Android. Internal Tracking ID: EXPOSUREAPP-6946